### PR TITLE
[Promotion] Make default rule checkers more generic

### PR DESCRIPTION
**Pull request by stloyd**

### CODE REVIEW

**Comment by stloyd** [comment]
This PR also fixes misplaced translations, but I'm not sure I should change other languages, @pjedrzejewski @aRn0D @umpirsky feedback?

File: src/Sylius/Bundle/CoreBundle/Form/Type/Rule/CustomerGroupType.php [hunk 1]
```diff
@@ -47,25 +45,18 @@ public function __construct(array $validationGroups, GroupRepository $groupRepos
...
-                    'query_builder' => function () use ($groupRepository) {
-                        return $groupRepository->getFormQueryBuilder();
-                    },
-                    'constraints' => array(
-                        new NotBlank(),
-                        new Type(array('type' => 'numeric')),
-                    ),
-                )
-            );
+        $builder->add('groups', 'sylius_entity_to_identifier', array(
```
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Form/Type/Rule/CustomerGroupType.php]: According to [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#44-method-arguments) previous method arguments formatting was more consistent.

Comment (stloyd) [inline src/Sylius/Bundle/CoreBundle/Form/Type/Rule/CustomerGroupType.php]: According to whole Sylius & Symfony code formating the new one is correct.

**Comment by pamil** [comment]
**Symfony**:
> Symfony follows the standards defined in the PSR-0, PSR-1, PSR-2 and PSR-4 documents.
[Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html)

**Sylius**:
> Sylius follows the standards defined in the PSR-0, PSR-1 and PSR-2 documents.
[Sylius Coding Standards](http://docs.sylius.org/en/latest/contributing/code/standards.html)

File: src/Sylius/Bundle/CoreBundle/Form/Type/Rule/CustomerGroupType.php (hunk 1 above)
Comment (pamil) [inline src/Sylius/Bundle/CoreBundle/Form/Type/Rule/CustomerGroupType.php]: **Symfony**:
> Symfony follows the standards defined in the PSR-0, PSR-1, PSR-2 and PSR-4 documents.
[Symfony Coding Standards](http://symfony.com/doc/current/contributing/code/standards.html)

**Sylius**:
> Sylius follows the standards defined in the PSR-0, PSR-1 and PSR-2 documents.
[Sylius Coding Standards](http://docs.sylius.org/en/latest/contributing/code/standards.html)

**Comment by pjedrzejewski** [comment]
I think we need to discuss all these naming conventions and make them generally shorter. This applies to all checkers/calculators/actions. Let's wait for improvements in Promotions from #3368.

**Pull request closed without merging**